istioctl/pkg/auth/testdata/rbac-policies-multiple-rules-v2.golden.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: rbac.istio.io/v1alpha1
+kind: AuthorizationPolicy
+metadata:
+  name: bind-service-viewer-0
+  namespace: default
+spec:
+  allow:
+  - role: service-viewer-0
```

Comment:
It would be nice if we can output some comments saying which `ServiceRole` this `AuthorizationPolicy` is generated from. We can some investigate later to see if this is possible.

Reply:
I'm writing a doc to explain the conversion process. This should help to reduce the confusion when customers are migrating to RBAC v2. We can also output a dump/log file to say where `service-viewer-0` is from.

Reply:
Wdyt?

Reply:
No need to output to a dump/log, it could just be part of the log. we will need the document anyway, adding to the converted yaml should be the easier way for people to understand the conversion.

Reply:
Sounds good I will add to the converted yaml

Reply:
Let's add the comments in the next PR. Wdyt?

Reply:
yes, that's what I was suggesting at first.